src/main/java/com/wolox/training/models/Book.java

```diff
@@ -0,0 +1,135 @@
+package com.wolox.training.models;
+
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column
+    private String genre;
+
+    @Column(nullable = false)
+    @NotNull
+    private String author;
+
+    @Column(nullable = false)
+    @NotNull
+    private String image;
+
+    @Column(nullable = false)
+    @NotNull
+    private String title;
+
+    @Column(name = "sub_title", nullable = false)
+    @NotNull
+    private String subTitle;
+
+    @Column(nullable = false)
+    @NotNull
+    private String publisher;
+
+    @Column(nullable = false)
+    @NotNull
+    private String year;
+
+    @Column(nullable = false)
+    @NotNull
+    private int pages;
+
+    @Column(nullable = false)
+    @NotNull
+    private String isbn;
+
+    public Book() {
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
```

Comment:
Remove the `setId`, we don't want the entity's id to be modified from the outside.

Reply:
Solved, i remove the setId method from the book model